android/sdk/src/main/java/org/jitsi/meet/sdk/PictureInPictureModule.java

```diff
@@ -44,14 +44,27 @@ public void enterPictureInPicture(Promise promise) {
             PictureInPictureParams.Builder builder
                 = new PictureInPictureParams.Builder()
                     .setAspectRatio(new Rational(1, 1));
-            boolean r
-                = currentActivity.enterPictureInPictureMode(builder.build());
+
+            boolean r = false;
+            Exception error = null;
+
+            // Quoting JSDoc for "enterPictureInPictureMode":
```

Comment:
Technically, the statement `Quoting JSDoc for "enterPictureInPictureMode"` is incorrect because that's a quote from the official Android documentation which is more likely to be javadoc than jsdoc.